### ISSUE TITLE
improvement

### ISSUE THREAD

**Issue by fabio1999ita** [aglarcz/D2R-lootfilter#1]
put sound when high item or rune drop and make this one more clickable like [here](https://github.com/AlexisEvo/d2r-loot-filter)

**Comment by aglarcz** [comment]
For me the goal of this filter is to reduce labels on the screen, mostly too long names of unwanted items.

I dont feel like adding anything to make it more like cheat. Using the mod online myself because I believe its okay to run visuals to reduce screen spam, thats another reason why I dont wanna do anything too deep. And making seperate, more advanced, version just for singleplayer is not something i have time for.

In other words I am going to change other items to shorter names but not interested in beams, sounds etc.

**Comment by fabio1999ita** [comment]
> mostly too long names of unwanted items.
> 
> I dont feel like adding anything to make it more like cheat.

first is not a cheat, second this two phrase contradict each other

**Comment by aglarcz** [comment]
Its not a cheat indeed but adding features which are not native to the game (drop sounds, beams etc.) is consider sort of cheating, atleast by me. 

And no "I dont feel like adding anything to make it more like cheat." doesnt not mean i will not add anything at all - it means i will not add anything i consider too aggresive as those features you suggested. As i wrote previously, one thing is to shorten item names, adding new features is another level and considered cheating by me.

**Comment by aglarcz** [comment]
Its not a cheat indeed but adding features which are not native to the game (drop sounds, beams etc.) is consider sort of cheating, atleast by me. 

And no "I dont feel like adding anything to make it more like cheat." doesnt not mean i will not add anything at all - it means i will not add anything i consider too aggresive as those features you suggested. As i wrote previously, one thing is to shorten item names, adding new features is another level and considered cheating by me.
